src/libraries/System.Net.Requests/ref/System.Net.Requests.cs

```diff
@@ -291,19 +293,19 @@ public partial class ProtocolViolationException : System.InvalidOperationExcepti
     {
```

Comment:
Above on row 281, 281 doc says `Authenticate( ... ) and PreAuthenticate(...)` methods can return null

Reply:
Yup.  I fixed the implementation but not the interface.  Will do.